frontend/src/static/js/v_zoom.js

```diff
@@ -1,3 +1,5 @@
+/* global Vuex */
```

Comment:
Why this comment is needed here?

Reply:
I believe it was a technique to avoid lint errors.

Reply:
OK then